app/js/mss/MssParser.js

```diff
@@ -167,7 +167,7 @@ Mss.dependencies.MssParser = function() {
             // Check if codec is supported
             if (SUPPORTED_CODECS.indexOf(fourCCValue.toUpperCase()) === -1) {
                 // Do not send warning
-                //this.errHandler.sendWarning(MediaPlayer.dependencies.ErrorHandler.prototype.MEDIA_ERR_CODEC_UNSUPPORTED, "Codec/FourCC not supported", {codec: fourCCValue});
+                this.errHandler.sendWarning(MediaPlayer.dependencies.ErrorHandler.prototype.MEDIA_ERR_CODEC_UNSUPPORTED, "Codec/FourCC not supported", {codec: fourCCValue});
```

Comment:
This not related with this PR. BTW we should remove this warning. We already put a warning in the console.